lib/rbvmomi/vim/Datastore.rb

```diff
@@ -63,6 +63,7 @@ def datacenter
   end
 
   def mkuripath path
-    "/folder/#{URI.escape path}?dcPath=#{URI.escape datacenter.name}&dsName=#{URI.escape name}"
+    datacenter_path_str = datacenter.path[1..-1].map{|elem| elem[1]}.join('/')
```

Comment:
If I'm reading this right, this will always back up one level ... look at the root folder then join the root folder and the child folder into a path... 

'NorthAmerica', 'MyDatacenter' becomes 'NorthAmerica/MyDatacenter' while 'MyDatacenter' becomes 'MyDatacenter' ... do I have that right?